kolibri/plugins/learn/assets/src/views/HomePage/ExploreChannels/index.vue

```diff
@@ -86,9 +86,22 @@
       },
     },
     mounted() {
-      // `crossComponentTranslator` needs to be used because it's after string freeze
       // TODO: Add 'View all' string into this component after 0.15 release
-      this.crossComponentStrings = crossComponentTranslator(YourClasses);
+      this.crossComponentStrings = createTranslator('YourClasses', {
```

Comment:
All strings on lines 91-103 can be moved to `$trs` object (line 119) which is a standard way of defining strings in our .vue files. `createTranslator` may be useful in some cases, for example in .js files. Please see https://kolibri-dev.readthedocs.io/en/develop/i18n.html. I didn't realize that it'd be useful to share this beforehand, apologies.

This is the case for other components changed in this PR too.

Reply:
So should I remove the `createTranslator` and instead directly use 
`this.crossComponentStrings = {
        name: 'YourClasses', 
        $trs: {
          yourClassesHeader: {
            message: 'Your classes',
            context: 'Refers to the classes the learner is enrolled in.',
          },
          noClasses: {
            message: 'You are not enrolled in any classes',
            context:
              'Message that a learner sees in the Learn > CLASSES section and in the Learn > HOME section if they are not enrolled in any classes.',
          },
          viewAll: {
            message: 'View all',
            context: 'Option to view all the classes the user is enrolled in.',
          },
        }
      };`
or should just use `$trs` object with `createTranslator` like this:
` this.crossComponentStrings = createTranslator('YourClasses',
      {
        $trs: {
          yourClassesHeader: {
            message: 'Your classes',
            context: 'Refers to the classes the learner is enrolled in.',
          },
          noClasses: {
            message: 'You are not enrolled in any classes',
            context:
              'Message that a learner sees in the Learn > CLASSES section and in the Learn > HOME section if they are not enrolled in any classes.',
          },
          viewAll: {
            message: 'View all',
            context: 'Option to view all the classes the user is enrolled in.',
          },
        }
      });` ?